recipe/build.sh

```diff
@@ -1,21 +1,25 @@
 #!/bin/bash
 
-# copy files which are missing from the release tarball
-# see: https://github.com/libssh2/libssh2/issues/379
-# TODO: remove this in the 1.9.1 or later releases
-cp ${RECIPE_DIR}/missing_files/*.c tests/
-
 # We use a repackaged cmake from elsewhere to break a build cycle.
 export PATH=${PREFIX}/cmake-bin/bin:${PATH}
```

Comment:
see this seems to be an line related to removed cmake-binary package

Reply:
These files used to be missing from libssh2 1.9.0 release. They are present in 1.10.0 release.

Reply:
well, that path variable looks to me superflous, isn't it?

Reply:
right, I removed the path setting.